app/src/main/java/com/gri/androidacademyworkshop/FragmentMoviesDetails.kt

```diff
@@ -2,20 +2,56 @@ package com.gri.androidacademyworkshop
 
 import android.os.Bundle
 import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.gri.androidacademyworkshop.adapters.ActorsAdapter
+import com.gri.androidacademyworkshop.model.Movie
+import com.gri.androidacademyworkshop.utils.Utils
 
-class FragmentMoviesDetails : Fragment(R.layout.fragment_movies_details) {
+class FragmentMoviesDetails(val movie: Movie) : Fragment(R.layout.fragment_movies_details) {
```

Comment:
leave fragment constructor empty
use arguments
https://stackoverflow.com/questions/9245408/best-practice-for-instantiating-a-new-android-fragment